config.json

```diff
@@ -642,9 +642,12 @@
       "slug": "diffie-hellman",
       "core": false,
       "unlocked_by": null,
-      "difficulty": 1,
+      "difficulty": 5,
       "topics": [
-
+        "Cryptography",
+        "Integers",
+        "Algorithms",
+        "Transforming"
```

Comment:
Honestly I'm torn on this one. For now I'm going to leave it as is. I think the conversation over having a `Standard library` topic is something separate.